.changeset/red-socks-camp.md

```diff
@@ -0,0 +1,8 @@
+---
+"@itwin/unified-selection": patch
```

Comment:
One of my comments suggests adding an extra optional prop to `computeSelection` - that means it should be a minor bump. Also, since it's an addition, it would be great to provide an extra explanation what the prop means and what its value should be set to. So I think we need a separate change file for the `@itwin/unified-selection` package.